node/core/pvf/src/lib.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+#![warn(missing_docs)]
+
+//! A crate that implements PVF validation host.
+//!
+//! This crate provides a simple API. You first [`start`] the validation host, which gives you the
+//! [handle][`ValidationHost`] and the future you need to poll.
+//!
+//! Then using the handle the client can send two types of requests:
+//!
+//! (a) PVF execution. This accepts the PVF [params][`polkadot_parachain::primitives::ValidationParams`]
+//!     and the PVF [code][`Pvf`], prepares (verifies and compiles) the code, and then executes PVF
+//!     with the params.
+//!
+//! (b) Heads up. This request allows to signal that the given PVF may be needed soon and that it
+//!     should be prepared for execution.
+//!
+//! The preparation results are cached for some time after they either used or was signalled in heads up.
+//! All requests that depends on preparation of the same PVF are bundled together and will be executed
+//! as soon as the artifact is prepared.
+//!
+//! # Priority
+//!
+//! PVF execution requests can specify the [priority][`Priority`] with which the given request should
```

Comment:
> [priority][`Priority`]

Is this valid syntax? :D

Reply:
Yeah I believe so, it's like Markdown but if the link destination is backticks then it will be resolved as an path.

Reply:
Okay fine :)

I always thought that only `[`Something`](something::Something)` works